draft-sharabayko-mops-srt.md

```diff
@@ -1201,9 +1201,7 @@ every control and data packet (see {{packet-structure}}).
 
 ## Data Transmission Modes {#data-transmission-mode}
 
-SRT has been mainly created for Live Streaming and therefore its main and
-default transmission mode is "live". SRT supports, however, the modes that
-the original UDT library supported, that is, buffer and message transmission.
+SRT supports the modes that the original UDT library supported, that is, buffer and message transmission. As SRT has been mainly created for Live Streaming, its main and default transmission mode is Message Mode with certain settings. Details of these settings are described in section Use Case Live Streaming {{use-case-live-streaming}}.
```

Comment:
This introduction should be improved in a way that:
- we clearly state that there are 2 data transmission modes supported by SRT: message and buffer (here we refer to appropriate subsections)
- then we tell that SRT was mainly designed for live streaming, etc.
- then we tell than file/message transmission is also possible via SRT and refer to "File/Message transmission Use Case" subsection which we currently don't have.